src/ng/rootScope.js

```diff
@@ -112,8 +112,24 @@ function $RootScopeProvider() {
      * compiled HTML template is executed.)
      *
      * Here is a simple scope snippet to show how you can interact with the scope.
-     * ```html
-     * <file src="./test/ng/rootScopeSpec.js" tag="docs1" />
+     * ```js
+         var scope = $rootScope.$new();
+         scope.salutation = 'Hello';
+         scope.name = 'World';
+
+         expect(scope.greeting).toEqual(undefined);
+
+         scope.$watch('name', function() {
+           scope.greeting = scope.salutation + ' ' + scope.name + '!';
+         }); // initialize the watch
+
+         expect(scope.greeting).toEqual(undefined);
+         scope.name = 'Misko';
+         // still old value, since watches have not been called yet
+         expect(scope.greeting).toEqual(undefined);
+
+         scope.$digest(); // fire all the watches
```

Comment:
I don't disagree regarding $digest not being recommended, but changing the example is not quite the scope of this request.  This example was accidentally removed so I am just putting it back. This sample comes directly from the unit tests. All the examples on this page use $digest; none of them use $apply.  The notes do indicate that $digest is not recommended, but similar notes regarding 'better practices' exist throughout the documentation.  If you would like to file an issue to have the examples all changed to $apply, I think that would be great and hopefully someone can pick it up.  Thanks!